package.json

```diff
@@ -177,7 +177,7 @@
     "file-loader": "6.2.0",
     "fork-ts-checker-webpack-plugin": "6.1.0",
     "fs-extra": "8.1.0",
-    "glob": "7.1.6",
+    "glob": "7.2.0",
```

Comment:
Necessary from the `lage` upgrade I assume?

Reply:
Not strictly necessary, but helpful to reduce duplication (one of the deps of updated `lage` introduced a dep on `glob@^7.2.0`) and very unlikely to break anything